src/main/java/org/resitrack/controller/PersonController.java

```diff
@@ -51,13 +94,76 @@ public void searchPeopleByName() {
         }
     }
 
+    /**
+     * This function is used to delete one or many people
+     *
+     * @param name
+     * @return true if delete process success and vice versa
+     */
+    public boolean deletePeopleByName(String name) {
+        List<Person> peopleByName = findPersonByName(name);
+        if (peopleByName.size() < 0) {
+            return false;
+        }
+        return deletePeople(peopleByName);
+    }
+
+    /**
+     * This function is used to show basic information of one person
+     *
+     * @param person
```

Comment:
please explain where is this param come from, what is value of it

Reply:
Đã update